monetio/models/_rrfs_sd_mm.py

```diff
@@ -0,0 +1,184 @@
+""" RRFS-CMAQ File Reader """
+import numpy as np
+import xarray as xr
+from numpy import concatenate
+from pandas import Series
+
+
+def can_do(index):
+    """
++    Determines if the given index can be processed.
++
++    Parameters:
++        index (int): The index to check.
++    
++    Returns:
++        bool: True if the index is the maximum value, False otherwise.
++    """
+    if index.max():
+        return True
+    else:
+        return False
+
+
+def open_mfdataset(
+    fname,
+    var_list=None,
+    surf_only=False,
+    **kwargs,
+):
+    # Like WRF-chem add var list that just determines whether to calculate sums or not to speed this up.
+    """Method to open RFFS-SD dyn* netcdf files.
+
+    Parameters
+    ----------
+    fname : string or list
+        fname is the path to the file or files.  It will accept hot keys in
+        strings as well.
+    surf_only: boolean
+        Whether to save only surface data to save on memory and computational
+        cost (True) or not (False).
+
+    Returns
+    -------
+    xarray.DataSet
+        RRFS-SD model dataset in standard format for use in MELODIES-MONET
+
+    """
+
+    # Read in all variables and do all calculations.
+    dset = xr.open_mfdataset(fname, concat_dim="time", combine="nested", **kwargs)
+    
+    if 'smoke' in dset.data_vars and 'dust' in dset.data_vars:
+        dset['PM25'] = dset.smoke + dset.dust
+        dset['PM25'].attrs['long_name'] = 'Particulate Matter < 2.5 microns'
+        dset['PM25'].attrs['units'] = "ug/kg"
+    if 'smoke' in dset.data_vars and 'dust' in dset.data_vars and 'coarsepm' in dset.data_vars: 
+        dset['PM10'] = dset.smoke + dset.dust + dset.coarsepm
```

Comment:
Might be better to use `pm10` for consistency. And currently in MM it would cause a problem for model var to have same name as obs var.